_clients/data-prepper/get-started.md

```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
 e51e700e-5cab-4f6d-879a-1c3235a77d18
 b4ed2d7e-cf9c-4e9d-967c-b18e8af35c90
 ```
+The remainder of this page shows examples for running from the Docker image. If you
+built from source, refer to [Developer Guide](developer_guide.md) for more information.
+
+However you configure your pipeline, you will run Data Prepper the same way. You run the Docker
+image and supply both the `pipelines.yaml` and `data-prepper-config.yaml` files.
+
+For Data Prepper 2.0 or above, use this command:
+
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
+```
+
+For Data Prepper before version 2.0:
+
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml opensearchproject/data-prepper:latest
```

Comment:
Data Prepper `latest` could be 2.0 soon. Perhaps we should change this to `data-prepper:1` which loads the latest from the 1.x series.